src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+                        = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createDriverStatement.setString(1, driver.getName());
+            createDriverStatement.setString(2, driver.getLicenseNumber());
+            createDriverStatement.executeUpdate();
+            ResultSet resultSet = createDriverStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create driver. " + driver.getName() + " ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT * FROM drivers WHERE id=? AND is_deleted = FALSE ";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriverStatement =
+                        connection.prepareStatement(query)) {
+            getDriverStatement.setString(1, String.valueOf(id));
+            ResultSet resultSet = getDriverStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(createDriver(resultSet));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get driver from DB by id "
+                    + id, throwables);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String query = "SELECT * FROM drivers WHERE is_deleted = FALSE ";
+        List<Driver> driverList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllDriversStatement =
+                        connection.createStatement()) {
+            ResultSet resultSet = getAllDriversStatement.executeQuery(query);
+            while (resultSet.next()) {
+                Driver driver = createDriver(resultSet);
+                driverList.add(driver);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get drivers from db", throwables);
+        }
+        return driverList;
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String query = "UPDATE drivers SET name=?, licenseNumber=? WHERE id=?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateStatement =
+                        connection.prepareStatement(query)) {
+            updateStatement.setString(1, driver.getName());
+            updateStatement.setString(2, driver.getLicenseNumber());
+            updateStatement.setLong(3, driver.getId());
+            updateStatement.executeUpdate();
+            return driver;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't update manufacturer: "
```

Comment:
copy-past is evil as you mention a manufacturer while we operate with a driver. would you please check all exceptions